src/events/dto/serialized-deposit.dto.ts

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+export interface SerializedDeposit {
```

Comment:
We use DTOs for payloads that our controllers accept (POST bodies). Can this file be renamed and/or moved to the `interfaces` folder?

Reply:
That's confusing as DTO (data transfer object) indicates that it should be for transfer out, and transfer in. This class is used as the response to a request. Does that not mean it is a schema for transfering an object?

Reply:
I moved it, though I'm still not sure the distinction. It seems like both `interfaces` and `dtos` contain DTO's, but DTO's are purely request input validators.

Reply:
Yeah, the current pattern is using DTOs for incoming request data objects that we can run validation on using `class-validator`. I'm open to moving the serializable return types (which omit fields from our models) if that seems more consistent with the DTO name, but I've usually seen them used for validation within NestJS.